weave

```diff
@@ -641,16 +653,20 @@ case "$COMMAND" in
             DNS_ARGS=$(exec_remote dns-args "$@")
             [ "$1" = "--with-dns" ] && shift 1
         fi
-        CIDR=$1
-        shift 1
-        validate_cidr $CIDR
+        CIDRS=""
+        while is_cidr "$1"; do
+            CIDRS="$CIDRS $1"
+            shift 1
+        done
```

Comment:
Just spent twenty minutes banging my head against this, wondering why it wasn't working - unfortunately the `$()` expression executes in a subshell, so the modified global `$CIDRS` is not visible to the calling process...